Sources/App/configure.swift

```diff
@@ -1,10 +1,20 @@
 import Vapor
+import Fluent
+import FluentPostgresDriver
+import Foundation
 
 // configures your application
 public func configure(_ app: Application) throws {
     // uncomment to serve files from /Public folder
     // app.middleware.use(FileMiddleware(publicDirectory: app.directory.publicDirectory))
 
     // register routes
+    app.migrations.add(AppUser.Migrations())
+    app.migrations.add(UserToken.Migrations())
+    try app.databases.use(.postgres(url: Application.db), as: .psql)
```

Comment:
Could use an in-memory sqlite database until you have the structures bottomed out, then they're re-created each time you run.

Reply:
Is the `autoMigrate` required here?

Reply:
good thinking @pdamonkey, i'll switch to that for debug?

Reply:
Sounds like a plan.  It's just handy while you're establishing all your migrations/models/seeding as it will create them all from scratch every time